tofu/geom/_core.py

```diff
@@ -3665,7 +3665,7 @@ def _prepare_inputs_kInOut(self, D=None, u=None, indStruct=None):
                 else:
                     num_tot_structs += len(ss.Lim)
 
-            lsnvert = np.asarray(lsnvert, dtype=np.int64)
+            lsnvert = np.asarray(lsnvert, dtype=np.long)
```

Comment:
Not sure how this affects linux users, waiting for feedback from Laura

Reply:
I'm gonna test it in my machine, could you try it also in CentOS ?